### PR TITLE
Reorganize Governance and README

### PR DESCRIPTION
**Pull request by josee-sabourin**
We have heard from community members that it is difficult to understand how to participate in GBFS development. MobilityData is re-evaluating the governance and trying to make it as easy as possible for someone to participate and to understand their role as a contributor as well as MobilityData's role as a maintainer. 

This first modification does not change any governance process (and therefore does not require a vote), it simply moves the governance to it's own page (governance.md) on the repository and shows a more simple version of the governance on the README.md. 

In the coming weeks we will work to improve the layout of governance.md and propose any changes to the governance if need be.

### CODE REVIEW

**Comment by mplsmitch** [comment]
I'm totally in favor of moving the governance to a separate document. The README needs frequent updates to things like current versions etc. that don't need to be voted on whereas any change to the governance should require a vote. Having the governance in a separate document will help with that.

I think the proposed README text could be more concise. There's some redundancy between the bullet points and the paragraph above. Here's what I would suggest:

> Comments or questions can be addressed to the community by [opening an issue](https://github.com/NABSA/gbfs/issues). Proposals for changes or additions to the specification can be made through [pull requests](https://github.com/NABSA/gbfs/pulls). Questions can also be addressed to the community via the [public GBFS Slack channel](https://bit.ly/mobilitydata-slack) or to the shared mobility staff at MobilityData: sharedmobility@mobilitydata.org. 
> 
> ### Proposing a Change 
> 
> - Anyone can propose a change to the specification by opening a pull request.
> - Proposed changes should align with the [Guiding Principles](https://github.com/MobilityData/gbfs#guiding-principles).
> - All changes are subject to a vote.
> - A proposal passes if there are at least 3 votes in favor and none opposed, with at least one vote from a GBFS producer, and one from a GBFS consumer.
> - Anyone can vote. 
> 
> If you would like to propose a change but aren’t comfortable opening a pull request, MobilityData is happy to help. Get in touch at [sharedmobility@mobilitydata.org](mailto:sharedmobility@mobilitydata.org).
> 
> For more information on the specification governance, including instructions on how and when to call a vote, see [governance.md](https://github.com/MobilityData/gbfs/blob/master/governance.md)
> 
> Find a real-world example of the governance in action [here](https://github.com/MobilityData/gbfs/pull/454).

File: README.md [hunk 1]
```diff
@@ -27,9 +26,19 @@ The specification has been designed with the following concepts in mind:
...
+
+## How to Participate
+GBFS is an open source project developed under a consensus-based governance model. Contributors come from across the shared mobility industry, public sector, civic technology and elsewhere. GBFS is not owned by any one person or organization. The specification is not fixed or unchangeable. As the shared mobility industry evolves, it is expected that the specification will be extended by the GBFS community to include new features and capabilities over time. <br/><br> Comments or questions can be addressed to the community by [opening an issue](https://github.com/NABSA/gbfs/issues). Proposals for changes or additions to the specification can be made through [pull requests](https://github.com/NABSA/gbfs/pulls). Questions can also be addressed to the community via the [public GBFS Slack channel](https://bit.ly/mobilitydata-slack) or to the shared mobility staff at MobilityData: [sharedmobility@mobilitydata.org](mailto:sharedmobility@mobilitydata.org).
```
Comment (richfab) [inline README.md]: Suggestion to replace `NABSA` repo by `MobilityData` repo.

```suggestion
GBFS is an open source project developed under a consensus-based governance model. Contributors come from across the shared mobility industry, public sector, civic technology and elsewhere. GBFS is not owned by any one person or organization. The specification is not fixed or unchangeable. As the shared mobility industry evolves, it is expected that the specification will be extended by the GBFS community to include new features and capabilities over time. <br/><br> Comments or questions can be addressed to the community by [opening an issue](https://github.com/MobilityData/gbfs/issues). Proposals for changes or additions to the specification can be made through [pull requests](https://github.com/MobilityData/gbfs/pulls). Questions can also be addressed to the community via the [public GBFS Slack channel](https://bit.ly/mobilitydata-slack) or to the shared mobility staff at MobilityData: [sharedmobility@mobilitydata.org](mailto:sharedmobility@mobilitydata.org).
```

File: README.md [hunk 2]
```diff
@@ -27,9 +26,19 @@ The specification has been designed with the following concepts in mind:
...
+* If you are wanting to open a pull request but don't know how, MobiilityData is happy to help. Get in touch at [sharedmobility@mobilitydata.org](mailto:sharedmobility@mobilitydata.org).
+* Discussions on pull requests must be a minimum of seven calendar days.
+* Votes are open for a total of 10 calendar days, anyone can vote.
+* A successful vote must have at least 3 votes, not including the pull request author.
+* A successful vote must include a vote from a GBFS producer and a GBFS consumer.
+
+Find a real-world example of the governance in action [here](https://github.com/MobilityData/gbfs/pull/454). For a more in depth look at the change and contribution process, go to [governance.md](https://github.com/MobilityData/gbfs/blob/master/governance.md).
 
 ### Project Roadmap
 MobiltyData has compiled a [project roadmap](https://github.com/NABSA/gbfs/wiki/Project-Roadmap) with a list of major features, changes and other work coming up in the near future.
```
Comment (richfab) [inline README.md]: Suggestion to replace `NABSA` repo by `MobilityData` repo.

```suggestion
MobiltyData has compiled a [project roadmap](https://github.com/MobilityData/gbfs/wiki/Project-Roadmap) with a list of major features, changes and other work coming up in the near future.
```

File: governance.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* The Advocate should cancel a vote and restart the process if significant changes are made to the proposal after stakeholders have voted.
+* Should the vote fail, The Advocate can choose to continue work on the proposal with the feedback received and restart the governance process, or abandon the proposal by closing the Pull Request. Another interested member of the community can take over the proposal if they feel the addition is valuable.
+* When a vote passes, the change is placed into Release Candidate (RC) status. The change remains in RC status pending implementation.
+    * Implementation requirements are that at least 1 producer and 1 consumer implement the changes.
+    * The implementors MUST be stakeholders other than The Advocate.
+    * Once implemented successfully, the change is merged into an official current release.
+* Editorial changes as well as items that are not found in [gbfs.md](https://github.com/NABSA/gbfs/blob/master/gbfs.md) do not need to be voted on. Extensions that include new capabilities and features MUST be voted on.
```
Comment (richfab) [inline governance.md]: Suggestion to replace `NABSA` repo by `MobilityData` repo.

```suggestion
* Editorial changes as well as items that are not found in [gbfs.md](https://github.com/MobilityData/gbfs/blob/master/gbfs.md) do not need to be voted on. Extensions that include new capabilities and features MUST be voted on.
```

File: README.md [hunk 4]
```diff
@@ -27,12 +26,22 @@ The specification has been designed with the following concepts in mind:
...
+If you are new to engaging with the community on this repository, firstly welcome! Here is a brief overview of how to contribute to the specification:
+* Anyone can raise an issue.
+* Anyone can open a pull request - make sure PRs in line with our [Guiding Principles](#guiding-principles).
+* If you are wanting to open a pull request but don't know how, MobiilityData is happy to help. Get in touch at [sharedmobility@mobilitydata.org](mailto:sharedmobility@mobilitydata.org).
+* Discussions on pull requests must be a minimum of seven calendar days.
```
Comment (richfab) [inline README.md]: Small suggestion for consistency with the line below.

```suggestion
* Discussions on pull requests must be a minimum of 7 calendar days.
```

**Review by richfab — CHANGES_REQUESTED** [review]
LGTM! Added one small formatting suggestion.

**Review by richfab — APPROVED** [review]
LGTM 🚀

**Pull request merged**